[IMP] email faqs: email limit (#9672)
odoo/documentation: content/applications/general/email_communication/faq.rst

Here are the default limits for new databases:

- **200 emails per day** for Odoo Online and Odoo.sh databases with an active subscription.
- **50 emails per day** for one-app free and trial databases.

> - In the case of migration, the daily limit might be reset to 50 emails per day.

Reviewer: Also, to fall in line with the formatting you've established up to this point, should the '50 emails per day' in this line be **bold**? Just something to consider during your revisions

.. important::
   In a one-app free database, if the *Email Marketing* app is installed, the email limit is **20
   emails per day**.

If the daily limit is reached:

> - Contact the Odoo support team, who may increase the daily limit depending on the following

Reviewer: ```suggestion
- Contact the Odoo support team, who may increase the daily limit, depending on the following
```
SUGGESTION: adding comma to enhance sentence pacing/flow

SMTP error
**********

> Simple Mail Transport Protocol (SMTP) error messages explain why an email was not transmitted

Reviewer: ```suggestion
*Simple Mail Transport Protocol (SMTP)* error messages explain why an email was not transmitted
```
suggesting the use of italics here because this is, essentially, a definition.

The debug menu can be used to investigate SMTP sending issues from a database. To access the menu,

> :ref:`developer mode <developer-mode>` must be activated. Once activated, navigate to the

Reviewer: ```suggestion
:ref:`developer mode <developer-mode>` **must** be activated. Once activated, navigate to the
```
OPTIONAL: adding more visual "importance" to this portion of the explanation

:menuselection:`Debug Menu` in the top-right of the menu bar (the :icon:`fa-bug` :guilabel:`(bug)`

> icon), :menuselection:`Debug Menu --> Manage Messages`.

Reviewer: ```suggestion
icon), and select :menuselection:`Debug Menu --> Manage Messages` from the resulting drop-down menu.
```

The :guilabel:`Manage Messages` menu opens a list of all the messages sent in a particular record.

> Within each message there is information on sending, including the type, and subtype, of the

Reviewer: ```suggestion
Within each message, there is information on sending, including the type, and subtype, of the
```

Email campaigns send at a scheduled time, using a delay pre-programed in the database. Odoo uses a
delayed task to send emails that are considered "not urgent" (newsletter formats, such as: mass
mailing, marketing automation, and events). The system utility **cron** can be used to schedule
programs to run automatically at predetermined intervals. Odoo uses that policy to avoid cluttering
the mail servers and, instead, prioritizes individual communication. This **cron** is called
:guilabel:`Mail: Email Queue Manager`, and can be accessed in :ref:`developer mode <developer-mode>`

> by going to :menuselection:`Settings app --> Technical menu --> Automation --> Scheduled Actions`.

Reviewer: ```suggestion
by going to :menuselection:`Settings app --> Technical menu --> Automation: Scheduled Actions`.
```
based on the feedback we discussed during the latest Writer's Workshop meeting, I *believe* this is how we decided to format such a direction moving forward

Incoming emails
===============

When there is an issue with incoming emails, there might not be an indication in Odoo. It is the

> sending email client, who tries to contact a database, that will get a bounce-back message (most of

Reviewer: ```suggestion
sending email client, who tries to contact a database, that gets a bounce-back message (most of
```

> the time a :guilabel:`550: mailbox unavailable` error message).

Reviewer: ```suggestion
the time it is a :guilabel:`550: mailbox unavailable` error message).
```

>    The bounce system parameter needs to be set in the technical settings in order for the database

Reviewer: ```suggestion
   The bounce system parameter needs to be set in the technical settings, in order for the database
```

   to correctly receive bounce messages. To access this setting, go to :menuselection:`Settings app

>    --> Technical menu --> Parameters --> System Parameters`. Then select the parameter name

Reviewer: ```suggestion
   --> Technical menu --> Parameters: System Parameters`. Then, select the parameter name
```

>    :guilabel:`mail.bounce.alias` and set the value to `bounce` if it is not already set.

Reviewer: ```suggestion
   :guilabel:`mail.bounce.alias`, and set the value to `bounce`, if it is not already set.
```
SUGGESTION: adding commas to enhance overall pacing/flow of the sentence
